app/templates/components/footer-main.hbs

```diff
@@ -40,6 +40,26 @@
           </a>
         </div>
       </div>
+    </div>
+  </div>
+  <div class="ui inverted section divider" style="margin-bottom: 55px!important;"></div>
+  <div class="ui stackable grid">
+    <div class="thirteen wide computer twelve wide tablet column left aligned">
+      <div class="ui horizontal inverted small divided link list no top margin">
+        <span class="item">{{t 'Copyright'}} &copy; {{moment-format (now) 'YYYY'}}  {{this.settings.appName}} </span>
+        <span class="hidden-item">
+          {{!-- Possible Page Values --}}
+          {{t 'Terms'}}
+          {{t 'Contact'}}
+          {{t 'Refund Policy'}}
+          {{t 'Privacy'}}
+        </span>
+        {{#each this.footerPages as |page|}}
+          <LinkTo @route="pages" @model={{page.url}} class="item">
+            {{t-var page.name}}
+          </LinkTo>
+        {{/each}}
+      </div>
```

Comment:
Why this much code added?

Reply:
i Don't there how was a merge conflict and it happened

Reply:
I only added style one